consensus/expected.go

```diff
@@ -265,18 +267,12 @@ func (c *Expected) RunStateTransition(ctx context.Context, ts types.TipSet, ance
 // validateMining checks validity of the block ticket, proof, and miner address.
 //    Returns an error if:
 //    	* any tipset's block was mined by an invalid miner address.
-//      * the block ticket is incorrectly computed
```

Comment:
I think this condition is separate from the one on line 270.  I would maybe phrase it as "* the block ticket is not a valid signature of the proof under the miner's signing key"

Reply:
block ticket checking is disabled for this PR anyway, unless you mean something else?